### PR TITLE
Added idp command to load functions from pdb file

### PR DESCRIPTION
**Pull request by xarkes**

### CODE REVIEW

File: libr/core/cmd_info.c [hunk 1]
```diff
@@ -447,7 +447,49 @@ static int cmd_info(void *data, const char *input) {
...
+				char* filename = strchr (input, ' ');
+				if (filename) {
+					*filename++ = '\0';
+					char* args = strchr (filename, ' ');
+					if (args) {
+						*args = '\0';
+					}
+					int current_fd = r_core_bin_cur (core)->fd;
+					char* r = r_core_cmd_strf (core, "o %s", filename);
+					int fd = atoi (r);
```
Comment (radare) [inline libr/core/cmd_info.c]: possible null deref if core_cmd_strf returns null

File: libr/core/cmd_info.c [hunk 2]
```diff
@@ -447,7 +447,49 @@ static int cmd_info(void *data, const char *input) {
...
+				}
+				char* filename = strchr (input, ' ');
+				if (filename) {
+					*filename++ = '\0';
+					char* args = strchr (filename, ' ');
+					if (args) {
+						*args = '\0';
+					}
+					int current_fd = r_core_bin_cur (core)->fd;
+					char* r = r_core_cmd_strf (core, "o %s", filename);
```
Comment (radare) [inline libr/core/cmd_info.c]: uhm, maybe better to use the R_IO api

File: libr/core/cmd_info.c [hunk 3]
```diff
@@ -447,7 +447,49 @@ static int cmd_info(void *data, const char *input) {
...
+					"Output mode:", "", "",
+					"'*'", "", "Output in radare commands",
+					"id", "", "Source lines",
+					"idp", " <file.pdb>", "Show pdb file information",
+					".idp*", " <file.pdb", "Load pdb file information",
+					NULL
+				};
+				r_core_cmd_help (core, help_message);
+			} else { // "id"
+					RBININFO ("dwarf", R_CORE_BIN_ACC_DWARF, NULL, -1);
```
Comment (radare) [inline libr/core/cmd_info.c]: bad indent

File: libr/core/cmd_info.c (hunk 2 above)
Comment (radare) [inline libr/core/cmd_info.c]: its ok for me to merge this using cmdstr, but should be done with the api if possible in another pr

File: libr/core/cmd_info.c [hunk 4]
```diff
@@ -447,7 +447,49 @@ static int cmd_info(void *data, const char *input) {
...
+					*filename++ = '\0';
+					char* args = strchr (filename, ' ');
+					if (args) {
+						*args = '\0';
+					}
+					int current_fd = r_core_bin_cur (core)->fd;
+					char* r = r_core_cmd_strf (core, "o %s", filename);
+					int fd = atoi (r);
+					free (r);
+					if (fd <= 0) {
```
Comment (radare) [inline libr/core/cmd_info.c]: fd = 0 is valid

**Comment by XVilka** [comment]
This one worth adding tests in the test suite as well

**Pull request merged**